src/style.css

```diff
@@ -0,0 +1,51 @@
+.star {
+  height: 12px;
+  width: 13px;
+  margin-right: 6px;
+  margin-bottom: 3px;
+  background-size: 100%;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--0 .star:nth-child(n) {
+  background-image: url("images/star.svg");
+}
+
+.stars--1 :nth-child(n + 1) {
+  background-image: url("images/star.svg");
+}
+
+.stars--1 :nth-child(1) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--2 :nth-child(n + 2) {
+  background-image: url("images/star.svg");
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--3 :nth-child(n + 3) {
+  background-image: url("images/star.svg");
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--4 :nth-child(n + 3) {
+  background-image: url("images/star.svg");
+}
+
+.stars--4 :nth-child(-n +  4) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--5 :nth-child(n) {
+  background-image: url("images/star-active.svg");
+}
```

Comment:
Let's make it even less complicated.

Apply background with empty star on default and then change it to active when necessary.
Also, you can combine selectors with same styles using `,`
```
selector1,
selector2,
selector3 {
  ...styles
}
```